src/main/resources/profiles/digital-commonwealth.conf

```diff
@@ -11,6 +11,5 @@ endpoint = "http://fedora.digitalcommonwealth.org/oaiprovider/"
 metadataPrefix = "mods"
 outputDir = "" # TODO This should be an S3:// bucket that i3 has write access to
 provider = "Digital Commonwealth"
-type = "oai" # Which harvester to invoke
```

Comment:
I removed it because we are not actively using it and didn't want to presume that this was going to be the implementation before we actually did it.